doc/classes/EngineDebugger.xml

```diff
@@ -0,0 +1,132 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<class name="EngineDebugger" inherits="Object" version="4.0">
+	<brief_description>
+		Exposes the internal debugger.
+	</brief_description>
+	<description>
+		[EngineDebugger] handles the communication between the editor and the running game. It is active in the running game. Messages can be sent/received through it. It also manages the profilers.
+	</description>
+	<tutorials>
+	</tutorials>
+	<methods>
+		<method name="has_capture">
+			<return type="bool">
+			</return>
+			<argument index="0" name="name" type="StringName">
+			</argument>
+			<description>
+				Returns true if a capture with the given name is present otherwise false.
+			</description>
+		</method>
+		<method name="has_profiler">
+			<return type="bool">
+			</return>
+			<argument index="0" name="name" type="StringName">
+			</argument>
+			<description>
+				Returns true if a profiler with the given name is present otherwise false.
+			</description>
+		</method>
+		<method name="is_active">
+			<return type="bool">
+			</return>
+			<description>
+				Returns true if the debugger is active otherwise false.
+			</description>
+		</method>
+		<method name="is_profiling">
+			<return type="bool">
+			</return>
+			<argument index="0" name="name" type="StringName">
+			</argument>
+			<description>
+				Returns true if a profiler with the given name  is present and active otherwise false.
+			</description>
+		</method>
+		<method name="profiler_add_frame_data">
+			<return type="void">
+			</return>
+			<argument index="0" name="name" type="StringName">
+			</argument>
+			<argument index="1" name="data" type="Array">
+			</argument>
+			<description>
+				Calls the [i]add[/i] callable of the profiler with given name with given data.
```

Comment:
For parameters, and code in general (unless it's a block) use the `[code][/code]` tag instead of `[i][/i]`